scripts/ingest.py

```diff
@@ -162,44 +164,52 @@ async def on_shutdown(server: IngestDeviceServer) -> None:
     server.halt()
 
 
-def main() -> None:
+async def get_async_telstate(endpoint: katsdptelstate.endpoint.Endpoint):
+    client = await aioredis.create_redis_pool(f'redis://{endpoint.host}:{endpoint.port}')
```

Comment:
What is the benefit of a Redis pool?

Reply:
If a connection dies you can get another one, and you can make requests in parallel using multiple connections.